lib/scan/port/engine.py

```diff
@@ -19,96 +20,24 @@
 from core.log import __log_into_file
 from core.compatible import is_windows
 from lib.payload.scanner.service.engine import discover_by_port
+from lib.scan.port import ports
+from scapy.config import conf
+from scapy.layers.inet import IP, TCP, ICMP
```

Comment:
changed
from scapy.all import *
to
appropriate imports wherever required in port/engine.py